### ISSUE TITLE
Bug: addon-a11y configuration not respected

### ISSUE THREAD

**Issue by kevinmpowell** [kalbeekatz/storybook-a11y-report#20]
## Bug description
When running storybook-a11y-report, storybook configuration for the addon-a11y plugin is not respected.

## Steps to reproduce
1. Create a storybook instance with a few example stories
2. Install addon-a11y and storybook-a11y-report plugins
3. In `.storybook/preview.ts|.js` add the following:
```
// .storybook/preview.ts

export const parameters = {
  a11y: { disable: true }
};
```
4. Run the storybook locally and confirm the a11y add-on is not running on any of the stories
5. Run the storybook-a11y-report test from the CLI and confirm that every story is still checked for accessibility.

## Expected behavior
storybook-a11y-report should respect any addon-a11y configuration present in the storybook including disabling/enabling of a11y globally or per story as well as disabling/enabling of individual a11y checks globally or per story as listed in under "Handling failing rules" in the [addon-a11y documentation](https://storybook.js.org/addons/@storybook/addon-a11y)

**Comment by kevinmpowell** [comment]
It looks like this line in the source should be taking story-level configuration into account before running tests:
https://github.com/kalbeekatz/storybook-a11y-report/blob/main/src/index.ts#L137

**Comment by kevinmpowell** [comment]
Confirmed: story level disabling of a11y rules is respected by this plugin.

From the a11y add-on docs:

```js
export const InputWithoutAutofill = () => <input type="text" autofill="nope" />;

InputWithoutAutofill.parameters = {
  a11y: {
    // Avoid doing this, as it will fully disable all accessibility checks for this story.
    disable: true,

    // Instead, override rules 👇
    // axe-core configurationOptions (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#parameters-1)
    config: {
      rules: [
        {
          // You can exclude some elements from failing a specific rule:
          id: 'autocomplete-valid',
          selector: '*:not([autocomplete="nope"])',
        },
        {
          // You can also signify that a violation will need to be fixed in the future
          // by overriding the result of a rule to return "Needs Review"
          // rather than "Violation" if the rule fails:
          id: 'landmark-complementary-is-top-level',
          reviewOnFail: true,
        },
      ],
    },
  },
};
```

See "Story-level overrides" [here](https://storybook.js.org/addons/@storybook/addon-a11y)

**Comment by kalbeekatz** [comment]
@kevinmpowell Thanks for reporting.

I fixed this issue v0.0.13.
I hope you continue to use story-a11y-report.
